wazo_call_logd/cel_interpretor.py

```diff
@@ -289,21 +303,52 @@ def interpret_wazo_user_missed_call(self, cel, call):
         ) = _extract_user_missed_call_variables(extra)
 
         if source_user_uuid:
-            source_participant = CallLogParticipant(
-                role='source',
+            call.source_user_uuid = source_user_uuid
+            info = dict(
```

Comment:
Stylistic preferences. less quotes.

Reply:
We try to keep same style everywhere by using only `{}`.
Moreover is easier for the reader to read `{}` its also perform better

Reply:
I'm skeptical of both claims? Why is it easier to read literal syntax with `"` and `:` symbols instead of keyword arguments syntax that is also extremely common in the language?

Reply:
The performance argument would matter in a tight loop, but I don't think it justifies a general policy.

Reply:
Admitted defeat, this was corrected.